pkg/controller/account/account_controller.go

```diff
@@ -87,24 +87,15 @@ func newReconciler(mgr manager.Manager) reconcile.Reconciler {
 		awsClientBuilder: &awsclient.Builder{},
 	}
 
-	configMap, err := utils.GetOperatorConfigMap(reconciler.Client)
-	if err != nil {
-		log.Error(err, "failed retrieving configmap")
-	}
-
-	hiveName, ok := configMap.Data["shard-name"]
-	if !ok {
-		log.Error(err, "shard-name key not available in configmap")
-	}
-	reconciler.shardName = hiveName
+	reconciler.shardName = config.GetHiveShardName()
```

Comment:
Is this change part of the ticket or just an extra to clean up the account controller a bit?

Reply:
this cleans up an error msg about the shardname not being present that shows while testing in crc